### DISCUSSION TITLE
SVG file type additionally

### DISCUSSION THREAD

**Discussion by laultman** [StereoKit/StereoKit#755] [Q&A]
Is there any chance to have SVG files also? We have a customer who has all their technical diagrams in SVG format (thousands of images).

**Comment by maluoi** [comment]
I have thought about this, and I'd certainly like it. Haven't found the time for it yet though, I expect this may be a lot more fun if I can find the right library for working with SVGs.

Can you give additional details about what requirements you'd need from SVGs? Like would you prefer they get rasterized to a Texture+Sprite, SDF vs regular raster, or turned into a Mesh+Model, or something else? What subset of functionality would you expect from an SVG (the whole SVG spec is almost certainly too large a project), etc.

**Comment by laultman** [comment]
@maluoi You can probably tell that I have not even so much as taken a glance at the SVG specs. 

My svg requirements are straight forward, I have a customer who has taken thousands of line drawings (B/W 2D) and rendered them as svg files. I need to simply display them as images same as I would a png file type.
Supplied a couple samples.

[sample-SVG.zip](https://github.com/StereoKit/StereoKit/files/13169758/sample-SVG.zip)

**Reply by maluoi** [reply]
Okay, thanks for the details! In the meantime, you can probably use a library like this one to convert them to raster, and the load them into a StereoKit Sprite or something:
https://github.com/svg-net/SVG

**Reply by laultman** [reply]
@maluoi  We got this project, thanks! We are putting it in the data-inline-workflow for the customer as a temporary procedure. The process does a conversion and stores a png file along with the svg. Works for now anyway. (new question coming in another topic)

**Comment by lewerentz** [comment]
Hello, 

it would be great to have svg added to the library!

**Reply by maluoi** [reply]
Hey thanks for the feedback! Do you think you could also weigh in on this? I'm still quite interested in how people are hoping to use the SVG format in their projects :)

> Can you give additional details about what requirements you'd need from SVGs? Like would you prefer they get rasterized to a Texture+Sprite, SDF vs regular raster, or turned into a Mesh+Model, or something else? What subset of functionality would you expect from an SVG (the whole SVG spec is almost certainly too large a project), etc.
